.envrc

```diff
@@ -1,3 +1,3 @@
 export NODE_VERSION_PREFIX=v
 export NODE_VERSIONS=~/.nvm/versions/node
-use node 20.12.2
```

Comment:
Yes, this uses nvm, but I use .direnv to load my environment configs since it can do more that just node version manager. Check this out here https://direnv.net/

Reply:
The `.nvmrc` file is commonly used, but direnv appears to offer even more capabilities. We better use a simple description for that.